usl_pipeline/study_area_uploader/tests/readers/test_polygon_readers.py

```diff
@@ -1,14 +1,21 @@
 import pathlib
 import pytest
 import tempfile
+from typing import Tuple
 
 import fiona
 import numpy.testing as npt
 import pyproj
+from shapely import geometry
 
 from study_area_uploader.readers import polygon_readers
 
 
+def _polygon_to_points(polygon: geometry.Polygon) -> list[Tuple[float, float]]:
+    xx, yy = polygon.exterior.coords.xy
+    return list(zip(xx, yy))
```

Comment:
Do callers need a list, or would they be fine with just the generator returned by `zip(xx, yy)`?
I don't see any callers of this function -- do we need it at all?

Reply:
Nope, I forgot to remove it. Good catch!